tox.ini

```diff
@@ -11,3 +11,15 @@ exclude=
     __pycache__,
     .tox,
     **data_messages_pb2.py
+
+[flake8]
+exclude=
+    build,
+    dist,
+    tests,
```

Comment:
Too much work right now but we should remove `tests` from the exclude list.

Reply:
Perhaps this should be a ticket?